bindings/python/iota_sdk/types/unlock_condition.py

```diff
@@ -131,3 +147,41 @@ class ImmutableAccountAddressUnlockCondition(UnlockCondition):
     address: AccountAddress
     type: int = field(default_factory=lambda: int(
         UnlockConditionType.ImmutableAccountAddress), init=False)
+
+
+def unlock_condition_from_dict(d: Dict[str, Any]) -> Union[AddressUnlockCondition, StorageDepositReturnUnlockCondition, TimelockUnlockCondition,
+                                                           ExpirationUnlockCondition, StateControllerAddressUnlockCondition, GovernorAddressUnlockCondition, ImmutableAccountAddressUnlockCondition]:
+    """
+    Takes a dictionary as input and returns an instance of a specific class based on the value of the 'type' key in the dictionary.
+
+    Arguments:
+    * `d`: A dictionary that is expected to have a key called 'type' which specifies the type of the returned value.
+    """
+    # pylint: disable=R0911
```

Comment:
we can write these disable statements as their explanation name
`too-many-return-statements`
Its a bit more readable